packages/Webkul/Shop/src/Listeners/Customer.php

```diff
@@ -24,25 +24,13 @@ public function afterCreated($customer)
                 if (! core()->getConfigData('emails.general.notifications.emails.general.notifications.verification')) {
                     return;
                 }
-
+                Mail::queue(new RegistrationNotification($customer, 'admin'));
                 Mail::queue(new EmailVerificationNotification($customer));
             } catch (\Exception $e) {
                 \Log::info('EmailVerificationNotification Error');
 
                 report($e);
             }
-
-            return;
-        }
-
-        try {
-            if (! core()->getConfigData('emails.general.notifications.emails.general.notifications.registration')) {
-                return;
-            }
-
-            Mail::queue(new RegistrationNotification($customer));
-        } catch (\Exception $e) {
-            report($e);
         }
```

Comment:
You need to add this check to your updated code. Because if we are not giving permission to send registration emails from the configuration, then we need to check here.

`core()->getConfigData('emails.general.notifications.emails.general.notifications.registration')`